.travis.yml

```diff
@@ -1,21 +1,27 @@
 language: python
 python:
 - '2.7'
+
 install:
 - pip install -r test-requirements.txt
 - pip install coveralls
 - pip install pyflakes==1.2.3
 - pip install pylint==1.5.6
 - pip install yapf==0.10.0
+- pip install google-compute-engine
+
 script:
 - ./build
 - $TRAVIS_BUILD_DIR/travis/install_gcloud.sh
+
 after_success:
 - coveralls
+
 before_install:
-- if [[ -n "$encrypted_ea5d7bea5193_iv" ]] ; then
-  travis/decrypt.sh "$encrypted_ea5d7bea5193_iv" "$encrypted_ea5d7bea5193_key"
-  keys/service-accounts.tar.enc /tmp/service-accounts.tar /tmp ; fi
+- $TRAVIS_BUILD_DIR/travis/decrypt.sh
```

Comment:
Since this is all one command, are line continuation characters (`\`) not required to join these 3 lines? Or is this some peculiarity of YAML and/or Travis-CI configs?

Reply:
This does work as-is. Similar examples are in prometheus-support and scraper's .travis.yml files.

My understanding is that this is how YAML does line continuation, and travis-ci inherits that behavior since it is YAML. The `\` for line continuation is more of a shell-ism. 

I'd be open to adopting a convention where we don't line wrap commands in .travis files or create small support scripts that can be called without as many parameters so that they fit on a single line.